stock_reserve/tests/test_stock_reserve.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2021 Tecnativa - Carlos Roca
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import fields
+from odoo.tests import Form, common
+
+
+class TestStockReserve(common.TransactionCase):
+    def setUp(self):
+        super().setUp()
+        self.env["res.config.settings"].create(
```

Comment:
```suggestion
        self.env["res.config.settings"].create(
```
Initially, the tests were failing due to a warning:
'Creating a new warehouse will automatically activate the Storage Locations setting.'

I enabled 'Track product location in your warehouse'.

![image](https://github.com/user-attachments/assets/3ec8b4b5-4a6d-43ed-9e9c-8f90af0522b3)